src/Halogen/HTML/Events/Types.purs

```diff
@@ -33,6 +36,23 @@ type MouseEvent =
   , which :: Number
   )
 
+-- | Identifies the additional fields which are available on drag events.
+type DragEvent =
+  ( button :: Number
+  , detail :: Number
+  , relatedTarget :: HTMLElement
+  , clientX :: Number
+  , clientY :: Number
+  , screenX	:: Number
+  , screenY	:: Number
+  , ctrlKey	:: Boolean
+  , shiftKey :: Boolean
+  , altKey :: Boolean
+  , metaKey	:: Boolean
+  , which :: Number
+  , dataTransfer :: Nullable DataTransfer
```

Comment:
I am a little bit not happy with `Nullable` here, but it seems that they are supposed to map directly. Is there a nice way to transform it to `Maybe`?